docs/CONTRIBUTING.md

```diff
@@ -66,6 +80,20 @@ After you are able to successfully build JsDbg, please follow the steps below to
 
 8. The JsDbg server will launch and prompt you to select a browser. If the debugger is attached to one browser, it is recommended that you choose a different browser for JsDbg.
 
+#### GDB
```

Comment:
Can we use the same numbering style instructions (1., 2., 3., ...) that we use for WinDbg and the VS debugger?

Reply:
done